client/src/ui/organisms/AddNewLedgerRecord.modal.jsx

```diff
@@ -37,8 +38,14 @@ export const AddNewLedgerRecordModal = ({ open, onClose, type }) => {
         await queryClient.refetchQueries([LEDGER_QUERY]);
         await queryClient.refetchQueries([BUDGET_QUERY]);
         await queryClient.refetchQueries([SUMMARY_QUERY]);
+        (type === 'EXPENSE')?(enqueueSnackbar('Wydatek został zapisany', {variant: 'success'})): (enqueueSnackbar('Wpływ został dodany', {variant: 'success'}));
```

Comment:
To bardzo nietypowe formatowanie kodu. `Ternary operator` raczej jest wykorzystywany w połączeniu z `return` lub przypisaniem do zmiennej. Następmnym razem lepiej wykorzystać go w następujący sposób:

```suggestion
const notificationMessage =  type === 'EXPENSE' ? 'Wydatek został zapisany' : 'Wpływ został dodany';

       enqueueSnackbar(notificationMessage, {variant: 'success'})
```